docs/products/m3db/getting-started.rst

```diff
@@ -0,0 +1,28 @@
+Getting Started
+===============
+
+There are two M3-related services available in the `Aiven console <https://console.aiven.io>`_. Get started with an **M3DB** service; this is the part that stores data.
+
+.. note::
+   The M3 Aggregator service provides extra query functionality when performing complex queries against the distributed data set
+
+Choose your cloud provider and location to deploy to, then choose which plan to use.
+
+If you're just trying out M3DB, a single-node setup is available with our startup plans. This isn't recommended for production use however; for that you should try the 3-node clusters on the business plans, or a premium plan to meet your needs.
```

Comment:
Should we mention that you can upscale downscale when needed? Is this happening while the service is online for M3?

Reply:
I'm not sure about this so I didn't include it yet.

Reply:
Oh, you should be able to upscale, I should at least say that!